examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
> IS PAIR_POTENTIALS only valid in a configuration data container?

There is a error here.  It should be in the SP.

> Does the user need to edit the contents at runtime (i.e. can we always point into a file to load the pair potentials configuration)?

I don’t understand the question.    In general, the user can configure the `PAIR_POTENTIALS` for the SP according to his/her problem. 

> Could PAIR_POTENTIALS be a list of PairPotentials objects? This means that one can create a PAIR_POTENTIALS setup programmatically. There could be also a method that reads a pair_potentials.yml file and returns a list of PairPotential instaces

Definitely.    Our current approach was to just stick the description in some formatted string.  This is ugly and not-user-friendly to SimPhony users; however its a temporary solution to get a working simple lammps-md up an running.   Next step is to figure out how to do this in a better way for the user.